### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by aliabbani**
## Blog app - setup and controllers

In this milestone I:

- Created a new app. Give it a meaningful name.
- Initialized your project with Git.
- I made sure that the project has the Postgres database set up.
- I made sure that the app handles the URLs used in the given wireframes.
- For each URL I created:
A route.
Action in the correct controller.
A view file.

Also, I set up:
ROR linters.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
-  get 'pages/hello'
-  root 'pages#hello'
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
+  root 'users#index' 
+  resources :users do 
+    resources :posts 
+ end
```
Comment (Dipeshtwis) [inline config/routes.rb]: - [ ] We can rewrite our routes in a more RESTFUL way to achieve the desired URLs mentioned in the project requirements. When I run `rails routes` the result of the URLs I get do not match the ones provided in the project requirements. See below:

URLs from project requirements | URLs from project
:---:|:---:
 | Screenshot from 2022-01-12 20-56-57

Please consider rewriting the routes as below:

```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

File: test/controllers/posts_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  test 'should get index' do
+    get posts_index_url
+    assert_response :success
+  end
+
+  test 'should get show' do
+    get posts_show_url
+    assert_response :success
+  end
+end
```
Comment (Dipeshtwis) [inline test/controllers/posts_controller_test.rb:13]: - I appreciate you wrote the test as well, as it's not required for this milestone. :clap: :clap:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @aliabbani, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] I would recommend making the README professional by adding a proper title, description, steps to clone and run the project. You can follow [this template](https://github.com/microverseinc/readme-template) as well.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @aliabbani,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**